docs/api/radio-group.md

```diff
@@ -28,11 +26,8 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 />
 
 
+A radio group is a container for a group of [radio buttons](./radio). It allows a user to select at most one radio button from a set. Checking one radio button that belongs to a radio group unchecks any previous checked radio button within the same group.
```

Comment:
Could we add a note along the lines of "for `radio-group` usage examples, see the `radio` page"?